rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -84,17 +90,17 @@ impl std::convert::From<Oneway> for bool {
 #[derive(Copy, Clone, Debug)]
 pub enum Infer<T> {
     None,
+    Guessed(T),
     Default(T),
```

Comment:
Maybe add some comments/examples of how Guessed differs from default?

I guess something like:

Guessed: no safe default value exists, so a potentially incorrect value was assumed in the absence of any relevant tags.
Default: A safe default value exists, and was used in the absence of any relevant tags.